tests/helper_functions.py

```diff
@@ -37,3 +39,26 @@ def add_bundle(store: Store, bundle: dict) -> None:
         """Add and commit bundle to housekeeper store"""
         bundle_obj, _ = store.add_bundle(bundle)
         store.add_commit(bundle_obj)
+
+    @staticmethod
+    def create_bundle_data(case_id: str, files: List[dict], created_at: dt.datetime = None) -> dict:
+        """
+        Create a new bundle_data dictionary with the given parameters.
+
+        :param case_id: The name of the bundle.
+        :param files: A list of dictionaries representing file data.
+        :param created_at: The timestamp when the bundle was created (optional).
+        :return: A dictionary representing the bundle data.
+        """
+        if created_at is None:
+            created_at = dt.datetime.now()
+
+        files = [{"path": str(file_data["file"]), "archive": True, "tags": file_data["tags"]} for file_data in files]
+
+        data = {
```

Comment:
should likely become a pydantic module

Reply:
I will refactor it in a separate PR anyway, I'm leaving it as is for now.